cxl/memdev.c

```diff
@@ -4448,6 +4526,77 @@ static int action_cmd_ddr_page_select_get(struct cxl_memdev *memdev,
 	return cxl_memdev_ddr_page_select_get(memdev);
 }
 
+static int action_cmd_ddr_hppr_set(struct cxl_memdev *memdev,
```

Comment:
hppr is set for both ddr0 and ddr1, can we have optional parameter to set for required ddr channel?

Reply:
On vistara side, we do have that control. If needed we can extend here. But kept single control for both controllers for  simplicity.